sale_partner_selectable_option_extended/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model_create_multi
```

Comment:
This code isn't actually `multi`, as return variable for `super` call is treated as a single record